identity/backend/src/main/java/io/camunda/identity/usermanagement/model/Group.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.identity.usermanagement.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "groups")
+public class Group {
+  @Id private Long id;
+
+  @Column(name = "group_name")
```

Comment:
💭 As we're in the `groups` table we can just leave this as `name` IMO

Reply:
Well, I kept the default Spring ddl so we don't need to change UserDetailsManager default implementation or set custom query for it.